### DISCUSSION TITLE
Academic Reverification Required

### DISCUSSION THREAD

**Discussion by ghost** [community/community#83959] [GitHub Education] [answered]
### Hi everyone,

I got an email saying I needed to reverify but when I go on the website not only does it say I'm approved but it doesn't let me reapply
![Capture](https://github.com/community/community/assets/149324955/377939c1-b843-4d6e-b007-924048e981a5)

**Comment by samus-aran** [comment, marked as answer]
Hi @Alex-Bastola 

Check if you have previously created GitHub accounts with your academic email. I suspect you have received that email because another coupon had expired. 

Please be aware that creating multiple free personal GitHub accounts is discouraged and is against our [terms of service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements). 
However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

**Reply by ghost** [reply]
Hey Samus-Aran, ended up getting my student benefits now. I think you are right because I had a github account associated with my student email in the past but it has since been deleted.

**Reply by jfenton888** [reply]
I have just come across this same problem as well. I had previously created a Github account with my university email as a second account to my personal one (a course required me to create one separate from my personal account), but deleted it so that I could apply for benefits as a part of my regular Github account. I have run into the exact same issue where the page that is meant to be linked in the email [https://education.github.com/discount_requests/application](https://education.github.com/discount_requests/application) does not allow me to fill it out again, as it was just completed a month ago. How did OP end up solving this?
<img width="659" alt="Screenshot 2024-03-03 at 4 44 10 PM" src="https://github.com/community/community/assets/30242958/bb3faa91-110e-4351-bd21-81a612a50203">
